apps/studio/data/database-extensions/database-extension-enable-mutation.ts

```diff
@@ -0,0 +1,71 @@
+import { useMutation, UseMutationOptions, useQueryClient } from '@tanstack/react-query'
+import { toast } from 'react-hot-toast'
+
+import { post } from 'data/fetchers'
+import { ResponseError } from 'types'
+import { databaseExtensionsKeys } from './keys'
+
+export type DatabaseExtensionEnableVariables = {
+  projectRef: string
+  connectionString: string
+  schema: string
+  name: string
+  version: string
+  cascade?: boolean
+}
+
+export async function enableDatabaseExtension({
+  projectRef,
+  connectionString,
+  schema,
+  name,
+  version,
+  cascade = false,
+}: DatabaseExtensionEnableVariables) {
+  let headers = new Headers()
+  if (connectionString) headers.set('x-connection-encrypted', connectionString)
+
+  const { data, error } = await post('/platform/pg-meta/{ref}/extensions', {
+    params: {
+      header: { 'x-connection-encrypted': connectionString },
+      path: { ref: projectRef },
+    },
+    body: { schema, name, version, cascade },
+    headers: Object.fromEntries(headers),
```

Comment:
Same comment as in `apps/studio/data/database-extensions/database-extensions-query.ts`